packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -106,14 +106,14 @@ export class GalleryContainer extends React.Component {
       );
     }
 
-    if (this.props.domId) {
-      this.currentHoverChangeEvent.domId = this.props.domId;
+    if (this.props.id) {
+      this.currentHoverChangeEvent.galleryId = this.props.id;
```

Comment:
This is internal to that gallery?

Reply:
Tes, cause itemView already had "id" as a prop (the actual idem id), so to itemView we pass the "id" of the gallery as "galleryId".
itemView is the part of the code that is actually using this event and the event data.